sycl/test/matrix/query-use.cpp

```diff
@@ -0,0 +1,143 @@
+// RUN: %clangxx -DSYCL_EXT_ONEAPI_MATRIX_VERSION=2 -fsycl -o query-use %s
```

Comment:
do we have a testcase in llvm-test-suite and check the query result?

Reply:
> do we have a testcase in llvm-test-suite and check the query result?

https://github.com/intel/llvm-test-suite/pull/1358 adds the test

Reply:
Ping @yubingex007-a11y